P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1023,164 +1023,365 @@ overshadows the historical content</quote>.</egXML>
+          </specList>
+	  The most relevant suggested values of <att>place</att> for
+	  ruby text are <val>above</val>, <val>below</val>,
+	  <val>left</val>, and <val>right</val>.</p>
+        
+        <p>In its simplest representation, a glossed form consists of an <gi>rb</gi> (ruby base) element containing the
+          base form, an <gi>rt</gi> (ruby text) element containing the gloss, and a <gi>ruby</gi> element which
+          wraps them together:
+          <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <ruby>
+              <rb>大学</rb>
+              <rt place="above">だいがく</rt>
+            </ruby>
+          </egXML>          
+          In the example above, the word <mentioned>大学</mentioned>
+          (<mentioned>daigaku</mentioned> = university) is provided
+          with a phonation gloss in hiragana. The full gloss is
+          applied to the complete word. However, it might instead be
+          broken down by character:
+          <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <ruby>
+              <rb>大</rb>
+              <rt place="above">だい</rt>
+            </ruby>
+	    <ruby>
+              <rb>学</rb>
+              <rt place="above">がく</rt>
+            </ruby>
+          </egXML>
+          Here is a similar example from Taiwan using bopomofo (pinyin
+          <mentioned>píngzi</mentioned> = bottle)<note
+          place="foot">Taken from <ref
+          target="https://en.wikipedia.org/wiki/Bopomofo#Example">Wikipedia</ref>.</note>:          
+          <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <ruby>
+              <rb>瓶</rb>
+              <rt place="right">ㄆㄧㄥˊ</rt>
+            </ruby>
+            <ruby>
+              <rb>子</rb>
+              <rt place="right">˙ㄗ</rt>
+            </ruby>
+          </egXML>
+         
+          
+          Where <att>place</att> is not provided, the default assumption is that the 
+          ruby gloss is <emph>above</emph> where the text is horizontal, and to the
+          <emph>right</emph> of the text where it is vertical. See <ptr target="#WDWM"/> for 
+          a detailed guide to writing modes and text directionality.
+          
+        </p>
+        
+        <p>The same ruby base may be accompanied by more than one gloss. 
+          Here, the Japanese word <mentioned>打球場</mentioned> (dakyūba, or <gloss>billiard hall</gloss>)
+          is glossed with  two different pronunciations: <q>biriādo</q> (its English equivalent)
+          and <q>dakyū</q>, a phonation guide for the first two characters.
+          
+          <figure>
+            <head><mentioned>Billiard hall</mentioned> with two ruby glosses. 
+              <ref target="http://school.nijl.ac.jp/kindai/NIJL/NIJL-01116.html#37">国文学研究資料館所蔵：：英国／龍動新繁昌記</ref>.</head>
+            <graphic url="Images/billiardhall.jpg" width="149px" height="276px"/>
+          </figure>
+          
+          This example is intriguing in that the right-side ruby 
```

Comment:
intriguing: "arousing one's curiosity or interest; fascinating." I think it means exactly what I think it means, in this case.

Reply:
I was not trying to suggest that “intriguing”  _means_ the wrong thing, only that it doesn’t sound right. Pretentious would be too strong a word to describe my complaint, but it’s on the right track. But if you like that word, and don’t think it harder for non-native readers, how about making the sentence a little more approachable by swapping “because” in for “in that”?

Reply:
The phrase "in that" appears 43 times in the Guidelines, so I think if it's likely to be confusing, then we have to change a lot of other locations. I generally favour simpler prose, but I don't think the current formulation constitutes the sort of impenetrable Britishism we used to have too much of. Perhaps what you're responding to is the fact that it makes the assertion that something is interesting at all -- maybe that's not for the Guidelines to say, and we should instead have "In this example, the right-side ruby...". Would that be better?

Reply:
Hmmm … this is tweak that, if needed, should not delay merging.